recipes/sentencepiece/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "sentencepiece" %}
+{% set version = "0.1.86" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: a8744e5d2943112e24b69a7ec895d8ea26c41bb55464b78af9c396d10fb3e02a
+
+build:
+  number: 0
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - pkgconfig
+    - gperftools
+  host:
+    - pip
+    - python
+    - pkgconfig
+    - gperftools
+    - pkg-config
+    - libcxx
```

```suggestion
    - pip
    - python
    - gperftools
```
(I suspect you don't need the pkg-config python bindings, if you you should re-add `pkgconfig` here)